core/tests/protractor/subscriptions.js

```diff
@@ -22,14 +22,19 @@ var PreferencesPage = require('../protractor_utils/PreferencesPage.js');
 var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
+var SubscriptionDashboardPage =
+  require('../protractor_utils/SubscriptionDashboardPage.js');
```

Comment:
Put `require` on previous line and break after '('

```
    var SubscriptionDashboardPage = require(
      '../protractor_utils/SubscriptionDashboardPage.js');
```

Reply:
when i done this test says 'error  Infix operators must be spaced  space-infix-ops'